docs/modules/user-guide/partials/proc_developing-with-devfiles.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc_developing-with-devfiles_{context}"]
+= Developing with devfiles
+[role="_abstract"]
+
+Develop a node.js “Hello World” application, using the devfile specification. Developing this application introduces you to how a devfile automates and simplifies your development process.
+
+.Prerequisites
+
+* To develop an application by using the devfile specification, you need to be on a cluster. To create a cluster, install link:https://minikube.sigs.k8s.io/docs/start/[minikube start].
+* To access your namespaces, download link:https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/[kubectl].
+* To execute the devfile specification, install the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/cli_tools/developer-cli-odo#installing-odo[odo cli tool].
+
+.Procedure
+
+. Build your Kubernetes cluster by running `minikube start`.
+. When you use minikube for the first time, you must enable Ingress in order to access your devfile projects. Enable Ingress by running `minikube addons enable ingress`.
+. Confirm you are on the default namespace by running `kubectl get namespaces`.
+. View the available devfiles by running `odo catalog list components`.
+. Create a devfile project by running `odo create nodejs <name of your project> --starter`.
+* Adding `--starter` tells odo to include the starter project currently inside the nodejs devfile specification, making it easier for you to develop an application.
```

Comment:
```suggestion
* Adding `--starter` tells odo to include the starter project currently inside the nodejs devfile specification, which makes it easier for you to develop an application.
```
- edit to avoid dangling gerund